rust/connlib/tunnel/src/tests/sim_client.rs

```diff
@@ -593,6 +590,7 @@ pub(crate) fn ref_client_host(
         any_ip_stack(),
         any_port(),
         ref_client(tunnel_ip4s, tunnel_ip6s),
+        latency(2000), // Clients might have a horrible Internet connection.
```

Comment:
@jamilbk You'll be pleased to know that we can handle latencies of up to 2000ms on clients and things still work.